### ISSUE TITLE
Clarification of the `config` example

### ISSUE THREAD

**Issue by jwreep** [rice-solar-physics/pydrad#172]
I'd like to suggest that the following snippet of the docs should be clarified a bit.  It's not obvious to a few collaborators what the "clean" copy means, or how this relates to setting up simulations.  I think the standard practice of most people who run HYDRAD is to make a copy of the directory for each simulation, but this isn't patently obvious to new users.

It would also be helpful to show an example where it points to a local copy, rather than just the download from Github.

--------------------

<img width="795" alt="Screenshot 2024-10-09 at 2 03 16 PM" src="https://github.com/user-attachments/assets/749fe7d5-b23b-49f0-8083-d98daef84dcf">

**Comment by wtbarnes** [comment]
That's a good point. I can definitely see why this is confusing. 

This is a semi-nonstandard usage and is mostly useful for running in a CI environment. We could just add a note in this example, but I wonder if it would be useful to add a separate section in the docs as well that gives a bit more background as to why things are done this way.

I wonder too if the function `get_clean_hydrad` should be renamed to something a little more informative. The usage of "clean" here implies that there are "dirty" HYDRADS as well! 😆

**Comment by jwreep** [comment]
Yes, I think having explanations of a few different ways would certainly be helpful.  There are also lots of options (numerics and physics), so a brief tutorial on changing things like radiative losses, or AMR options, etc., might also be useful.  I think we've listed the variables in the docs, but a plot of, for example, the emissivity for different radiation options would really hammer home what they mean.
